### PR TITLE
chore(RHTAPWATCH-641): collect unsigned RPMs

### PR DESCRIPTION
**Pull request by avi-biton**
implement the get_unsigned_rpms method
add unit tests

### CODE REVIEW

File: tests/test_rpm_verifier.py [hunk 1]
```diff
@@ -7,24 +7,61 @@
...
+            [],
+            id="All signed",
+        ),
+        pytest.param([], [], id="Empty list"),
+    ],
+)
+def test_get_unsigned_rpms(test_input: list[str], expected: list[str]) -> None:
+    """Test get_unsigned_rpms"""
+    mock_runner = MagicMock()
+    mock_runner.return_value.stdout.splitlines.return_value = test_input
```
Comment (yftacherzog) [inline tests/test_rpm_verifier.py:45]: `splitlines` is a method of the str. I think we should mock the return value of `stdout` and then test our actual usage of `splitlines`.

File: verify_rpms/rpm_verifier.py [hunk 2]
```diff
@@ -26,8 +26,31 @@ def get_rpmdb(container_image: str, target_dir: Path, runner: Callable = run) ->
 
 
 def get_unsigned_rpms(rpmdb: Path, runner: Callable = run) -> list[str]:
-    """Get unsigned RPMs from RPM DB path"""
-    raise NotImplementedError()
+    """
+    Get unsigned All RPMs from RPM DB path
```
Comment (yftacherzog) [inline verify_rpms/rpm_verifier.py:30]:
```suggestion
    Get all unsigned RPMs from RPM DB path
```

File: tests/test_rpm_verifier.py (hunk 1 above)
Comment (avi-biton) [reply tests/test_rpm_verifier.py]: Done

File: verify_rpms/rpm_verifier.py (hunk 2 above)
Comment (avi-biton) [reply verify_rpms/rpm_verifier.py]: Done

**Review by yftacherzog — APPROVED** [review]
lgtm

**Pull request merged**